### PR TITLE
Android style guide

### PR DESCRIPTION
**Pull request by murtazakhussain**

### CODE REVIEW

**Comment by mohuk** [comment]
@murtazakhussain thanks for the awesome work!

Just a few points:
- Instead of providing all the setup guidelines on the README.md file, shouldn't we just point to [this](https://developer.android.com/studio/write/lint.html) link
- rename `10PLint.xml` to simply `lint.xml` similar to the guide in the link above

Pointing to their guide will free us from the hassle of updating this if the configuration changes.

**Comment by murtazakhussain** [comment]
> Instead of providing all the setup guidelines on the README.md file, shouldn't we just point to this link

Yes, we can also do that, but there are additional steps also mentioned in the Readme file, but we can also include the link as well. 

> rename 10PLint.xml to simply lint.xml similar to the guide in the link above

Naming "10PLint" helps in identifying Lint structure to apply on the code in Android Studio. The default name was "Project Default"

**Comment by murtazakhussain** [comment]
`rename 10PLint.xml to simply lint.xml similar to the guide in the link above`

Lint file has been renamed.

File: android/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Icon problems (like missing densities, duplicate icons, wrong sizes, etc)
+- Usability problems (like not specifying an input type on a text field)
+- Manifest errors
+
+### How to import 10PLint.xml in Android Studio
+
+1. File -> Settings
+2. Editor -> Inspection 
+3. In "Manage" dropdown -> Import
+4. Select "10PLint.xml" file
```
Comment (ubaid-me) [inline android/README.md:22]: 10pLint.xml file does not exist now.

Comment (murtazakhussain) [reply android/README.md:22]: it has been renamed to lint.xml

Comment (ubaid-me) [reply android/README.md:22]: I know that's why I want you to change this ReadMe as well  😄

Comment (murtazakhussain) [reply android/README.md:22]: updated :)

Comment (mohuk) [reply android/README.md:22]: @murtazakhussain done with the changes?

Comment (murtazakhussain) [reply android/README.md:22]: yes.

**Pull request merged**